Source/DafnyCore/Resolver/ResolutionErrors.cs

```diff
@@ -8,9 +8,77 @@ namespace Microsoft.Dafny;
 public class ResolutionErrors {
 
   public enum ErrorId {
-    // ReSharper disable once InconsistentNaming
```

Comment:
Don't we still need this?

Reply:
Perhaps - but I don't know what it means.